### ISSUE TITLE
Integration with GA4

### ISSUE THREAD

**Issue by andersonferbr** [KaterinaLupacheva/react-google-analytics-dashboard#9]
Does this component allow integration with Google Analytics 4? Because in GA4, the "viewId" property no longer exists. How can we replace this data?

**Comment by KaterinaLupacheva** [comment]
@andersonferbr No, it doesn't work with GA4. There are breaking changes in GA4. Should be a completely different implementation.

**Comment by shuhad** [comment]
> @andersonferbr No, it doesn't work with GA4. There are breaking changes in GA4. Should be a completely different implementation.

Can you please work on this. a new version that works on GA4 please
